docs/plugins/testing_workshop_docs/index.md

```diff
@@ -0,0 +1,5 @@
+This is the frontpage for the testing workshop articles. 
```

Comment:
We will need to update this as this file should serve as the "landing page" for this set of articles/tutorials. Some of the information in the Assert keyword article should be moved here, including links to the video, where the example plugin is for users to get started and other general bits of information.

Reply:
I believe I have done this.  Please look at it and see what you think.  Good suggestion.